youtube_dl/extractor/playtvak.py

```diff
@@ -0,0 +1,86 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..compat import compat_urlparse
+
+
+class PlaytvakIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:.*\.)?(playtvak|idnes|lidovky|metro)\.cz/.*\?c=(?P<id>.*)'
```

Comment:
Do not capture group if you don't use it. `id` must match at least one character.